src/access_nri_intake/cli.py

```diff
@@ -27,7 +27,7 @@ class MetadataCheckError(Exception):
     pass
 
 
-def _parse_build_inputs(config_yamls, build_path):
+def _parse_build_inputs(config_yamls, build_path, data_path):
```

Comment:
I think it might make more sense to rename `data_path` => `data_base_path` here? As far as I can see it's only called once & uses `data_base_path` in that call - might be easier to track in future keeping that variable name?